### PR TITLE
Mitschriften SWE 09.11.2020

### PR DESCRIPTION
**Pull request by RvNovae**

### CODE REVIEW

**Comment by Steve2955** [comment]
## Command results
<details>
<summary>Details: </summary>

<details>
<summary><em>Run doctoc</em></summary>

```Shell
changed:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/SWE/README.md
unchanged:
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/DBS/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/DBS/praktikum/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/PRO/README.md
  - /home/runner/work/dhge-pi19-sem3/dhge-pi19-sem3/RES/README.md
```



</details>

</details>

## Changed files
<details>
<summary>Changed file: </summary>

- SWE/README.md

</details>

<hr>

[:octocat: Repo](https://github.com/technote-space/toc-generator) | [:memo: Issues](https://github.com/technote-space/toc-generator/issues) | [:department_store: Marketplace](https://github.com/marketplace/actions/toc-generator)

File: SWE/README.md [hunk 1]
```diff
@@ -455,4 +465,86 @@ Allgemeine Kriterien:
...
+> ein Epic ist ein höheres Anforderungsniveau -> keine Details -> nicht sofort umsetzbar  
+> Der Backlog enthält <= 150 Epics (wenn mehr -> zwei Produkte (Empfehlung))  
+> in der Regel besteht ein Epic aus 1-8 User-Stories
+- Der Product-Owner ist für die "Vision" und die Reihenfolge der Epics verantwortlich
+- Jeder soll/darf Epics in das Backlog "füttern" dürfen
+
+## User-Story
+> Als ... möchte ich ... um ...
+
+- Erklärung eines Fteaures aus Sicht einer Person, die dieses Feature verwenden will
```
Comment (ZeroPointMax) [inline SWE/README.md:495]: Features*

**Review by ZeroPointMax — CHANGES_REQUESTED** [review]
ein kleiner Rechtschreibfehler, ansonsten sehr schöne Mitschriften :-) 👍

File: SWE/README.md (hunk 1 above)
Comment (RvNovae) [reply SWE/README.md:495]: Oh ja, danke haha

**Review by ZeroPointMax — APPROVED** [review]
👍

**Pull request merged**